src/benchmarks/real-world/ImageSharp/Program.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Running;
+using System.IO;
+
+namespace SixLabors.ImageSharp.Benchmarks
+{
+    public class Program
+    {
+        /// <summary>
+        /// The main.
+        /// </summary>
+        /// <param name="args">
+        /// The arguments to pass to the program.
+        /// </param>
+        public static void Main(string[] args) => BenchmarkSwitcher
```

Comment:
Do you need our `Program.cs`?

Reply:
For now, yes. That's how we have our other real-world benchmarks setup.